src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1649,17 +1714,7 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
         throw xrt_core::error("The 'quick' value for the tests to run cannot be used with any other name tests.");
 
       // Verify the current user test request exists in the test suite
-      bool nameFound = false;
-      for (auto &test : testNameDescription) {
-        if (userTestName.compare(test.first) == 0) {
-          nameFound = true;
-          break;
-        }
-      }
-
-      if (nameFound == false) {
-        throw xrt_core::error((boost::format("Invalid test name: '%s'") % userTestName).str());
-      }
+      doesTestExists(userTestName, testNameDescription);
```

Comment:
```suggestion
      doesTestExist(userTestName, testNameDescription);
```
If we are validating only one test I think doesTestExist is correct

Reply:
I believe this is correct, if we were testing multiple test it would have been "doTestsExist"